src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -21,10 +22,17 @@
 
 (def valid-commands-str (str/join ", " (sort (vals command-names))))
 
+(defn maybe-str->number
+  [s]
+  (if (number? s) s (edn/read-string s)))
```

Comment:
the edn reader is an awfully heavy hammer to bring in here. Would Integer/parseInt do?